assets/packages/expressions/em_javascript.js

```diff
@@ -139,9 +139,6 @@ function fixnum_checkconditions(value, name, type, evt_type, intonly)
         }
 
         var addition = "";
-        if(cleansedValue && cleansedValue.split("").pop().match(/(,)|(\.)/)){
```

Comment:
Not sure why this was here on the first place.

Reply:
:+1: